app/build.gradle

```diff
@@ -78,6 +78,7 @@ android {
                     mapsApiKey: System.getenv('ANDROID_MAPS_API_KEY') != null ? System.getenv('ANDROID_MAPS_API_KEY') : android_maps_api_key
             ]
             buildConfigField("String", "GA_TRACKING_ID", "\"${ga_tracking_id_debug}\"")
+            buildConfigField "String", "CONFERENCE_TIMEZONE", "\"Asia/Tokyo\""
```

Comment:
it is preparation of the next function that user can show session time as local time.